helm/auth_service/apis/initialise/onOrchResponse.js

```diff
@@ -78,6 +60,16 @@ module.exports = function(responseObj, request, forwardToMS, sendResponse, getJW
 
       console.log('openehr response: ' + JSON.stringify(openehrResponse, null, 2));
 
+      // is DDS configured for use?
+      if (!globalConfig.DDS || !globalConfig.DDS.enabled) {
```

Comment:
@killmenot , are you sure that we should use globalConfig variable, as I see you defined global_config, is it correct?

Reply:
@DmitrySolyannik You're correct